### PR TITLE
[15.0][MIG] stock_barcodes_picking_batch: Migration to v15.0

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT45429

Standard migration to v15.

dependens on:

- [x] https://github.com/OCA/stock-logistics-barcode/pull/495

supersed https://github.com/OCA/stock-logistics-barcode/pull/523

ping @carlosdauden @CarlosRoca13 @Tisho99

### CODE REVIEW

**Review by carlosdauden — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Please remove latest commit.

**Comment by sergio-teruel** [comment]
Rebased and removed the latest commit

File: stock_barcodes_picking_batch/tests/test_stock_barcodes_picking_batch.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+from odoo.tests.common import tagged
+
+from odoo.addons.stock_barcodes.tests.test_stock_barcodes_picking import (
+    TestStockBarcodesPicking,
+)
+
+
+@tagged("post_install", "-at_install")
+class TestStockBarcodesPickingBatch(TestStockBarcodesPicking):
+    def setUp(self):
```
Comment (rousseldenis) [inline stock_barcodes_picking_batch/tests/test_stock_barcodes_picking_batch.py]: Prefer classmethod instead.

**Review by rousseldenis — APPROVED** [review]
Code review. Minor change

File: stock_barcodes_picking_batch/tests/test_stock_barcodes_picking_batch.py (hunk 1 above)
Comment (sergio-teruel) [reply stock_barcodes_picking_batch/tests/test_stock_barcodes_picking_batch.py]: Upps!!! +1

Comment (pedrobaeza) [reply stock_barcodes_picking_batch/tests/test_stock_barcodes_picking_batch.py]: And please include the context keys for speeding up tests.

Comment (sergio-teruel) [reply stock_barcodes_picking_batch/tests/test_stock_barcodes_picking_batch.py]: Done!

**Pull request merged**